setup.py

```diff
@@ -20,10 +20,9 @@
     name="cs3client",
     version=version,
     author="Rasmus Welander, Diogo Castro, Giuseppe Lo Presti",
-    package_dir={"": "src"},
-    packages=find_packages(where="src"),
-    py_modules=["cs3client"],
+    packages=find_packages(),
     description="CS3 client for Python",
+    package_dir={"cs3client": "cs3client"},
```

Comment:
This should not be needed, if you keep the convention of putting the code inside of a folder with the same name of the module.

Reply:
I have tested this a bit now and it seems that if i do not include this the files are scattered all over site-packages/ and then duplicated inside site-packages/src/, so for now I will include it.